packages/canner-wrapper/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "canner-wrapper",
```

Comment:
@wwwy3y3 @abz53378 I think this package should name `@canner/wrapper` how do you think? Just like babel 7 every pkgs prefix with `@babel/xxx`

Reply:
babel 6 every pkg prefix with `babel-xxx`, in babel 7 it changes to `@babel/xxx`, I think we should follow the trend.

Reply:
Changing naming is ok, but we have to change all packages' s right? Or just `canner` and `canner-wrapper`, how do you think?

And I currently only can figure out `container`, `wrapper`, `owner`, `controller`.